jax/experimental/compilation_cache/compilation_cache.py

```diff
@@ -20,11 +20,12 @@
 
 _cache = None
 
-def initialize_cache(path):
-    """Creates a global cache object. Should only be called once per process."""
+def initialize_cache(path, max_cache_size_bytes=32 * 2 **30):
+    """Creates a global cache object. Should only be called once per process.
+       max_cache_sixe is set to 32GiB."""
```

Comment:
Nit: Change the docstring format slightly, fix typo, and also mention it _defaults_ to 32GiB:
```python
    """Creates a global cache object. Should only be called once per process.

       max_cache_size defaults to 32GiB.
    """
```

Also mega nit, remove nit between 2 and **: `32 * 2**30`